assets/armsnippets.jsonc

```diff
@@ -1940,6 +1940,24 @@
         ],
         "description": "Public IP Prefix"
     },
+    "Resource Group": {
+        "prefix": "arm-rg",
+        "body": [
+            "{",
+            "\t\"name\": \"${1:resourceGroup}\",",
+            "\t\"type\": \"Microsoft.Resources/resourceGroups\",",
+            "\t\"apiVersion\": \"2019-10-01\",",
+            "\t\"location\": \"[parameters('location')]\",",
+            "\t\"dependsOn\": [",
+            "\t],",
+            "\t\"tags\": [",
+            "\t],",
+            "\t\"properties\": {",

```

Comment:
nit for consideration; properties are not necessary here and can be removed for brevity.